### DISCUSSION TITLE
Welcome to usbscope50 Discussions!

### DISCUSSION THREAD

**Discussion by phjr** [phjr/usbscope50#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeremysword** [comment]
I have the sdk and all the source code for the usbscope50 if you want it. Where should i upload? I was able to install on windows 10 x64 just had to disable driver signing requirement.

**Reply by peterhalicky** [reply]
I also have it, no need to get it again :-) but thanks for the offer.

**Comment by broken-brick440** [comment]
Thank you for maintaining the project.
I was able to create a VCP driver inf file using Silicon Labs' "CustomUSBDriverWizard" for the USB driver and install it on USBscope50. Then, I followed the Microsoft instructions to generate a .cat from the .inf and self-sign it, which allowed Java to run on Secure Boot Win11. The Java application appears to be working fine.
Can I share the steps here?

**Reply by peterhalicky** [reply]
By all means, please do. Alternatively if you like you can create a new README.md (the existing one is no good) with all the instructions -- and submit a pull request or a patch.

**Reply by peterhalicky** [reply]
I actually managed to fix the link in the README.md, so now it points to the snapshot of the old page with instructions and discussions.

**Reply by broken-brick440** [reply]
Thank you for saving the old discussion page. I submitted a pull request. However, I forgot to write the test environment. Would it be better to add it?

**Reply by peterhalicky** [reply]
PR merged, thank you! What do you mean by 'test environment'?

**Reply by broken-brick440** [reply]
Test environment information is Windows 11 23H2 SecureBoot, USB 3.0 Port, 4 stack configuration, etc. It is very impressive that it works even with the latest OS!

**Reply by peterhalicky** [reply]
Ah OK, I see. I think it's OK as-is for now, no need to add these details in the readme.
